docs/src/main/sphinx/develop/supporting-merge.md

```diff
@@ -233,14 +233,15 @@ The merge case `RowBlock` has the following layout:
 - A block containing the `MERGE` case operation number, encoded as
   `INSERT` = 1, `DELETE` = 2, `UPDATE` = 3 and if no `MERGE`
   case matched, -1.
-- A block containing the number, starting with 0, for the
+- A block containing the `MERGE` case number, the number starting with 0, for the
   `WHEN` clause that matched for the row, or -1 if no clause
   matched.
 
 The page returned from `transformPage` consists of:
 
 - All table columns, in table column order.
 - The merge case operation block.
+- The merge case number block.
```

Comment:
Can you add the data types of all the 3 special blocks?

Reply:
Please also update the `ConnectorMergeSink API` section, and make sure to describe what the connector is supposed to do with the new merge case block.